src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md

```diff
@@ -114,12 +114,16 @@ To ensure that the text you add in a `.js` file is collected by the i18n tool an
 
    ```javascript
    $.mage.__('<string>');
+   (or)
+   $t('<string>');
```

Comment:
Hi @anandhan-ziffity. There's a minor nuance in this statement. In the previous list item we are linking the translate library in the following way:

```javascript
   define (['jquery', 'mage/translate'], function ($) {...});
   ```

In order to use `$t` alias, we should change the import part to the following one 

```javascript
   define (['jquery', 'mage/translate'], function ($, $t) {...});
```